tests/unit/conftest.py

```diff
@@ -52,11 +52,21 @@ def nginx_container():
                     "mimirtool",
                     "rules",
                     "sync",
-                    f"--address=http://{socket.getfqdn()}:8080",
+                    f"--address=http://{socket.getfqdn()}:{NGINX_PORT}",
                     "--id=anonymous",
                 ],
                 return_code=0,
-            )
+            ),
+            Exec(
+                [
+                    "mimirtool",
+                    "rules",
+                    "sync",
+                    f"--address=https://{socket.getfqdn()}:{NGINX_TLS_PORT}",
+                    "--id=anonymous",
+                ],
+                return_code=0,
+            ),
```

Comment:
Nit: if you store the addresses in vars, e.g. `http_arg = f"--addres=..."` then perhaps the code will fold into something nicer.